cmd/clone.go

```diff
@@ -84,6 +84,23 @@ func clone(ctx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("abs of %s: %s", dst, err)
 	}
+
+	srcMp, err := findMountpoint(srcAbsPath)
+	if err != nil {
+		return err
+	}
+	dstMp, err := findMountpoint(filepath.Dir(dstAbsPath))
```

Comment:
no need, just check if dstAbsPath has srcMp as the prefix

Reply:
It doesn't apply in this case
```
src=/tmp/mp1/dir1
dst=/tmp/mp10/dir2
```